lints/lint_issuer_multiple_rdn.go

```diff
@@ -0,0 +1,45 @@
+package lints
+
+import (
+	"encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zcrypto/x509/pkix"
+	"github.com/zmap/zlint/util"
+)
+
+type IssuerRDNHasMultipleAttribute struct {
+	// Internal data here
+}
+
+func (l *IssuerRDNHasMultipleAttribute) Initialize() error {
+	return nil
+}
+
+func (l *IssuerRDNHasMultipleAttribute) CheckApplies(c *x509.Certificate) bool {
```

Comment:
Does this really apply to all certificates and not just subscriber certificates?